cert_manager/datadog_checks/cert_manager/data/conf.yaml.example

```diff
@@ -0,0 +1,38 @@
+init_config:
+
+instances:
+
+    ## @param prometheus_url - string - required
```

Comment:
This file does not look up to date. I'm wondering why validation is not failing for it

Reply:
ah it's because it is not being recognized by ddev
```
cert_manager:
    No spec found
```

Reply:
@davidor Can you please add this code block under "assets" in the manifest.json?

```
 "configuration": {
      "spec": "assets/configuration/spec.yaml"
    },
```